frontend/packages/ceph-storage-plugin/src/components/bucket-class/create-bc.scss

```diff
@@ -15,7 +15,6 @@
 
 .nb-create-bc-step-page-form__dropdown {
   display: block;
-  height: 20rem;
```

Comment:
Does it affect the original wizard page dropdowns ?

Reply:
No it doesn't seem to affect those when creating a bucket class.